src/mlops_codex/base.py

```diff
@@ -48,7 +48,7 @@ def __init__(
         if url is None:
             url = os.getenv("MLOPS_URL")
         if url is None:
-            url = "https://neomaril.datarisk.net/"
+            url = "https://mlops.datarisk.net/"
```

Comment:
The backend is still neomaril.datarisk.net

Reply:
- https://github.com/datarisk-io/mlops_codex/pull/68/commits/e31e9dfa1604edcf0d27b76b1239df78ccfe37f7
- https://github.com/datarisk-io/mlops_codex/pull/68/commits/51ef22a6582845c2bdc90b429854f2b8b188e5ad
- https://github.com/datarisk-io/mlops_codex/pull/68/commits/872497ff6bb3d6fb0d6e6add57ae4d31b91c283a